### ISSUE TITLE
Zoom-adaptive smoothing

### ISSUE THREAD

**Issue by pachuco** [drawpile/Drawpile#308]
More often then not, when at zooms above 100%, the smoothing option proves to be nothing short of a pain in the arse, while still being a convenience in terms of line quality.
It would be nice if stroke smoothing adapted to current zoom level.

**Comment by Wade821** [comment]
Fixed/added in the beta client. Feel free to comment there if you have any further concerns.

Latest version available here:
https://github.com/drawpile/Drawpile/releases

Last release on website (out of date):
https://drawpile.net/download/#Beta

**Comment by pachuco** [comment]
I am thinken, how this would work is that, smoothing happens at viewport's level instead of canvas level.

**Comment by askmeaboutlo0m** [comment]
Yes, it makes sense conceptually, it's just not implemented. A far as I can tell no other drawing program does it that way, so we can't use anything else as a known working base either, meaning this will probably just stay open for now.
